app/views/questionnaire.py

```diff
@@ -276,13 +333,32 @@ def submit_answers(routing_path, eq_id, form_type, collection_id):
         if not sent:
             raise SubmissionFailedException()
 
+        submission_validity = 0
+        submitted_time = int(time.time())
+
+        if store_submitted_responses():
+            submission_validity = g.schema_json['view_submitted_responses'].get('timeout', EQ_SUBMISSION_VALIDITY)
+            encrypter = StorageEncryption(current_user.user_id, current_user.user_ik)
+            encrypted_data = encrypter.encrypt_data(
+                {
+                    'answers': answer_store.answers,
+                    'metadata': metadata
+                }
+            )
+
+            SubmittedResponses().insert_item(metadata['tx_id'], encrypted_data, submitted_time, submission_validity)
```

Comment:
I think we should look at if we can extend `EncryptedQuestionnaireStorage` and provide multiple implementations then we can just update the storage mechanism on `QuestionnaireStore` and save a copy?
This would also set us in a good place for if we want to use DynamoDB for storing answers eventually